### PR TITLE
Added change password card to profile panel.

### PR DESCRIPTION
**Pull request by jeradM**
Backend PR: https://github.com/home-assistant/home-assistant/pull/15527

changepw

### CODE REVIEW

File: src/panels/profile/ha-dialog-change-password.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+          error-message='Required'
+        ></paper-input>
+        <paper-input
+          label='New Password'
+          type='password'
+          value='{{_newPassword}}'
+          required
+          auto-validate
+          error-message='Required'
+        ></paper-input>
```
Comment (awarecan) [inline src/panels/profile/ha-dialog-change-password.js]: We should ask user input new password twice, or allow user unmask the password, likes

https://github.com/home-assistant/home-assistant-polymer/blob/3f0824bd396478120602f8158f53fbcd6d5ae410/src/components/ha-form.js#L40-L55

File: src/panels/profile/ha-dialog-change-password.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+      // Error message when can't talk to server etc
+      _errorMsg: String,
+
+      _opened: {
+        type: Boolean,
+        value: false,
+      },
+
+      _currentPassword: String,
+      _newPassword: String,
```
[the remark this answers is not in the document]
Comment (jeradM) [reply src/panels/profile/ha-dialog-change-password.js]: So instead of a button to unmask the password?

[the remark this answers is not in the document]
Comment (balloob) [reply src/panels/profile/ha-dialog-change-password.js]: Let's not do an unmask button, that is more a login experience thing. I would just have 2 inputs: `_newPassword1` and `_newPassword2`

[the remark this answers is not in the document]
Comment (jeradM) [reply src/panels/profile/ha-dialog-change-password.js]: Okay, so no current password verification? Or keep that too?

[the remark this answers is not in the document]
Comment (balloob) [reply src/panels/profile/ha-dialog-change-password.js]: Keep the current password verification.

**Comment by balloob** [comment]
Also make sure to include a screenshot of how it looks in your PR description.

**Review by balloob — APPROVED** [review]

**Comment by pepeEL** [comment]
Please also add translation strings for this PR.

**Pull request merged**